include/pistache/cookie.h

```diff
@@ -73,18 +96,22 @@ class CookieJar {
 
     private:
         Storage::const_iterator it_;
+        HashMapCookies::const_iterator it_2;
```

Comment:
Please use a more descriptive name for these three private member variables.  Suggestion: "iter_storage", "iter_cookie_values", "iter_storage_end".